src/main/resources/templates/company.html

```diff
@@ -1,51 +1,46 @@
 <!DOCTYPE html>
-<html xml:th="http://www.thymeleaf.org">
+<html xmlns:th="http://www.thymeleaf.org" lang="en-US">
 <head>
-  <link rel="stylesheet" href="index.css">
+  <title>Voluntier</title>
+  <link rel="stylesheet" th:href="@{css/Index.css}">
+
   <script>
     function openForm() {
       document.getElementById("myForm1").style.display = "block";
+      document.getElementById("signUpButton").style.display = "none"
     }
 
     function closeForm() {
-      document.getElementById("myForm").style.display = "none";
+      document.getElementById("myForm1").style.display = "none";
+      document.getElementById("signUpButton").style.display = "block"
     }
   </script>
 </head>
+
 <body>
 <div class="content">
   <header>
-    <img src="images.png" alt="Logo">
-
+    <img th:src="@{/images/images.png}" alt="Logo">
   </header>
 
-
   <div class="login">
-
-
-
-    <!--/*@thymesVar id="volunteer" type=""*/-->
-    <p th:text="${company}">
-
-
+    <p th:text="${company}"></p>
```

Comment:
Adding end `</p>` tag for readability and to ensure HTML compliance in the future. While in this case you may omit the closing tag since it's followed immediately by a `<div>` tag, I would recommend closing it in case you add a tag in the future that does not allow you to omit the end `</p>` tag. If you want to stick with the current behavior, I recommend looking at the reference to determine when the end `</p>` tag is necessary. [Reference](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/p)